docs/source/overview.rst

```diff
@@ -15,6 +15,7 @@ Features
 
 * No need to specify enquiries period (there is no *active.enquiries* status), since it overlaps with *active.tendering* period.
 * `tenderPeriod` must be at least 7 calendar days.
+* Organizer can both increase and decrease `value.amount`, `guarantee.amount`, `minimalStep.amount`.
```

Comment:
One of the statements within the overview section is "Organizer can't edit procedure's significant properties (*Auction.value*, etc.)". This one needs removal.